### DISCUSSION TITLE
Splunk 8.2 issues

### DISCUSSION THREAD

**Discussion by mbinvitae** [mbegan/Okta-Identity-Cloud-for-Splunk#40] [Q&A]
Was this plugin tested on Splunk 8.2 (8.2.2 to be more specific)?
After 7.3 to 8.2 upgrade, we started seeing duplicate logs. We see >8-14 duplicate events (stats count by _raw) and also we are getting Okta API rate limit warning.

**Comment by mbinvitae** [comment]
@mbegan

**Reply by sushaw** [reply]
@mbegan 
Hi, The latest app is still not compatiable with Jquery and thus it cannot be used in Splunk Cloud SaaS installation. Below is the comments from Splunk Support team about this add-on 
"Unfortunately although the developer appeared to have covered all of the bases for jquery vulnerabilities, they seem to have missed some of the necessary work, as the latest version is incompatible.
If the developer needs assistance getting the app back into a compatible state, they can reach out to [appinspect@splunk.com](mailto:appinspect@splunk.com)  for assistance."
